### PR TITLE
docs: Create PH20xx.md

### PR DESCRIPTION
**Pull request by bcollamore**

### CODE REVIEW

File: Documentation/Diagnostics/PH2006.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+The following namespaces are always exempt from this rule. See supporting information for reasoning.
+
+| Namespace | Justification  |
+|--|--|
+| `System.Runtime.CompilerServices` | [Init setter support in .NET Standard 2.x](https://developercommunity.visualstudio.com/t/error-cs0518-predefined-type-systemruntimecompiler/1244809) |
+
+## Example
+
+If the file path is `.\Philips.CodeAnalysis.MaintainabilityAnalyzers\Naming\NamespaceMatchFilePathAnalyzer.cs` then the following code will pass for the default mode but fail in the more strict mode:
```
Comment (ynsehoornenborg) [inline Documentation/Diagnostics/PH2006.md:39]: I prefer to use obvious example names, as opposed to values that _could_ be confused with real names in the code.

**Review by ynsehoornenborg — APPROVED** [review]

File: Documentation/Diagnostics/PH2006.md (hunk 1 above)
Comment (bcollamore) [reply Documentation/Diagnostics/PH2006.md:39]: @ynsehoornenborg I don't understand can you elaborate?

Comment (bcollamore) [reply Documentation/Diagnostics/PH2006.md]: I think I understand. Updated example.

File: Documentation/Diagnostics/PH2001.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+| Diagnostic ID | PH2001 |
+| Category  | [Documentation](../Documentation.md) |
+| Analyzer | [XmlDocumentationShouldAddValueAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/XmlDocumentationShouldAddValueAnalyzer.cs)
+| CodeFix  | Yes |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+This analyzer checks if the text of the XML code documentation has content. Without content, it's not adding value. This analyzer is similar [PH2102](./PH2102.md).
```
Comment (ynsehoornenborg) [inline Documentation/Diagnostics/PH2001.md:15]: I would say "related" iso "similar"

**Review by ynsehoornenborg — DISMISSED** [review]

File: Documentation/Diagnostics/PH2001.md (hunk 2 above)
Comment (bcollamore) [reply Documentation/Diagnostics/PH2001.md]: updated

**Review by ynsehoornenborg — APPROVED** [review]

**Pull request merged**